yellow-paper/docs/decentralization/actors.md

```diff
@@ -0,0 +1,135 @@
+    - Requires large storage and a stable internet connection.
+    - Can be used to simulate transactions and view current state and provide history for new nodes to sync.
+3. **Archive Node**:
+    - Stores complete state, logs and history as well as historical state. 
+      - Supports historical state retrieval and membership proofs also known as storage proofs.
+    - Needs significant storage capacity.
+
+When running an Aztec node the actor must connect it to nodes of the underlying blockchain(s). The underlying chains being where data is published and proofs validated. For the following outline, I'm assuming that we just have 1 underlying chain, the Ethereum L1. However, in the future, if data is published on other chains, we will need to extend the constellations with those as well. Also, **blobs** are not stored forever, so what is usually referred to as the history of Ethereum will not longer contain them but just their commitments.
+
+#### Special nodes
+Special nodes are nodes that have specific tasks beyond synchronizing the chain. Namely they participate in block production. In Ethereum they are called validators, but for the sake of briefness, we skip those here, and mainly focus on the Aztec specific nodes.
+
+These nodes are the sequencer and prover nodes. To function they must be connected to a Pruned Full Node, Full Node or Archive Node. 
+
+1. **Sequencer**: 
+   - Requires computational power to simulate blocks.
+   - Needs a robust network to coordinate proof generation and propagate blocks.
+2. **Provers**: 
+   - Have high-performance CPUs/GPUs and a lot of memory.
+   - Assists in proving zero-knowledge (zk) proofs for block validation.
+
+### Aztec Data Retrieval Capabilities:
+An Aztec node needs to be connected to an Ethereum L1 node to learn about the state of the system. Initially synching can be done this way, but since the L1 can always be used as the "trust-anchor" for the state, an Aztec node can retrieve state from a third party and then validate it against the values of L1.
+
+As the Aztec node can be connected to different types of underlying network nodes, its ability to retrieve data will vary. Below we outline briefly what the capabilities are for each type of node:
+1. **Ethereum Light Node**:
+   - Relies on third party for getting L2 headers and block data.
+2. **Ethereum Pruned Full Node**:
+   - Can lookup recent L2 headers and blocks from own data.
+   - Relies on third party for getting historical blocks
+3. **Ethereum Full Node**:
+   - Can lookup L2 headers from own data.
+   - Can lookup recent L2 blocks from own data
+   - Relies on third party for getting historical blocks
+4. **Ethereum Archive Node**:
+   - Can lookup L2 headers and blocks from own data.
+
+### Actor Types
+
+1. **Mainstream User**:
+   - Least knowledgeable and least interested in technical details.
+   - Prefers simplicity and ease of use.
+   - Likely to avoid running any node software or only running if bundled
```

Comment:
I'd maybe break this further into users who are willing to make public queries to a 3rd-party node; and users who insist on making privacy-preserving queries to special 3rd-party nodes.